tests/bluetooth/tester/src/l2cap.c

```diff
@@ -19,11 +19,12 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "bttester.h"
 
 #define CONTROLLER_INDEX 0
-#define DATA_MTU 264
+#define DATA_BUF_SIZE (256 + BT_L2CAP_CHAN_SEND_RESERVE)
+#define DATA_MTU (DATA_BUF_SIZE - BT_L2CAP_CHAN_SEND_RESERVE)
```

Comment:
I realize this is starting to feel a bit like bike shedding, but wouldn't it have been more natural to do:
```
#define DATA_MTU       256
#define DATA_BUF_SIZE  (DATA_MTU + BT_L2CAP_CHAN_SEND_RESERVE)
```

Anyway, I'm keeping my "approved" so feel free to proceed as you wish